xwayland/xwm.h

```diff
@@ -0,0 +1,84 @@
+#ifndef XWAYLAND_INTERNALS_H
+#define XWAYLAND_INTERNALS_H
+#include <wayland-server-core.h>
+#include <wlr/xwayland.h>
+
+/* wlc's atom list:
```

Comment:
weston uses a struct with those atoms as members iirc, which makes more sense than what wlc does. Should not really be more complicated but way cleaner than those globals values.